bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/vehicle-status-channel-types.xml

```diff
@@ -11,15 +11,27 @@
 	<channel-type id="windows-channel">
 		<item-type>String</item-type>
 		<label>Overall Window Status</label>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="lock-channel">
 		<item-type>String</item-type>
 		<label>Doors Locked</label>
+		<tags>
+			<tag>Status</tag>
+			<tag>Mode</tag>
```

Comment:
Is `Mode` the appropriate property for lock status?

Reply:
Changed to `OpenState`